harvest/api/_base.py

```diff
@@ -255,7 +255,9 @@ def fetch_price_history(
         :param end: The ending date of the period, inclusive.
         :returns: A pandas dataframe, same format as main()
         """
-        raise NotImplementedError("This endpoint is not supported in this broker")
+        raise NotImplementedError(
+            f"{type(self).__name__} does not support this streamer method: `fetch_price_history`."
```

Comment:
type(self).__name__ will return the class of self and is useful when letting the user know what exact function has errored.